assigned-prs/assigned-prs.2m.sh

```diff
@@ -1,8 +1,8 @@
 #! /bin/bash
 
 ######### CONFIGURATION #########
-GITHUB_USER="" # e.g. "gabro"
-GITHUB_COM_TOKEN="" # get one at https://github.com/settings/tokens
+GITHUB_USER="raasoft" # e.g. "gabro"
+GITHUB_COM_TOKEN="258fbe2a1dd80ffc10a7d7c68c36361e08568e80" # get one at https://github.com/settings/tokens
```

Comment:
è un token vero?

Reply:
ahah si ho fatto un pasticcio stamattina, comunque l'idea era: ti va di svecchiare un po' questi script? perchè non usiamo più github enterprise e in generale usiamo un metodo di autenticazione che pare essere outdated (mi arrivano mail da GH)

Reply:
ok, ci sta, ma secondo me è un task alla portata di tutti quindi preferirei non farlo io.

Per i token, rimuovili poi (github te li invalida comunque, però meglio evitarE)

Reply:
comunque non ho capito la PR, di cosa devo fare review?

Reply:
ok, grazie.
In questo momento, volevo metterti al corrente di aver rimosso github enterprise e vedere se ti andava bene, dato che eri l'ultima persona che aveva committato quei files

Reply:
no ok, ero confuso perché quando mi hai messo in review la PR l'unica modifica era l'aggiunta delle tue credenziali. Comunque la PR è approvata ora